2023/gaussian-convolution/spanish/description.json

```
@@ -1,26 +1,26 @@
 [
  {
-  "translatedText": "Un truco visual para calcular la suma de dos variables distribuidas normalmente.",
+  "translatedText": "Un truco visual para calcular la suma de dos variables normalmente distribuidas.",
   "input": "A visual trick to compute the sum of two normally distributed variables."
  },
  {
-  "translatedText": "Lista de correo 3b1b: https://3blue1brown.substack.com/",
+  "translatedText": "Lista de correo de 3b1b: https://3blue1brown.substack.com/",
   "input": "3b1b mailing list: https://3blue1brown.substack.com/"
  },
  {
-  "translatedText": "Ayude a financiar proyectos futuros: https://www.patreon.com/3blue1brown",
+  "translatedText": "Ayuda a financiar proyectos futuros: https://www.patreon.com/3blue1brown",
   "input": "Help fund future projects: https://www.patreon.com/3blue1brown"
  },
  {
-  "translatedText": "Un agradecimiento especial a estos seguidores: https://www.3blue1brown.com/lessons/gaussian-convolution#thanks",
+  "translatedText": "Gracias especialmente a estos seguidores: https://www.3blue1brown.com/lessons/gaussian-convolution#thanks",
```

I do not follow many youtubers, but I swear that the few I occasionally watch would say "un agradecimiento especial" instead of "gracias especialmente".

```suggestion
  "translatedText": "Un agradecimiento especial a estos seguidores: https://www.3blue1brown.com/lessons/gaussian-convolution#thanks",
```

---

Suggestion committed [here](https://github.com/pabloasenjo3/captions/commit/08dc1e6edce68d406a78de3e525573641f88dd72).